README.md

```diff
@@ -13,6 +13,7 @@ SETUP ENVIRONMENT
 - Add MonoGame-templates according to: https://stackoverflow.com/questions/57879424/how-to-open-a-new-monogame-desktop-project-in-visual-studio-2019-community-editi)
 - Install Tiled 1.4.1 (https://thorbjorn.itch.io/tiled)
 - Install Git LFS
+	- You may also need to run `git lfs install`
```

Comment:
Don't you need to be in the repository when running this? Perhaps do this after the git clone step.